qa/nightwatch-sl.conf.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable */
+require('nightwatch-cucumber')({
+    cucumberArgs: [
+        '--require-module', 'babel-core/register',
+        '--require', 'features/step_definitions',
+        '--format', 'node_modules/cucumber-pretty',
+        '--format', 'json:reports/cucumber.json',
+        'features'
+    ]
+});
+
+module.exports = {
+    output_folder: 'reports',
+    page_objects_path: 'pages',
+    custom_commands_path: 'commands',
+    test_workers: {
+        enabled: true,
+        workers: 'auto' // number of cores used. "auto" means the same number as cores in the cpu
+    },
+    test_settings: {
+        default: {
+            launch_url: 'https://stillingsok-q.nav.no',
```

Comment:
Bruk gjerne 'https://arbeidsplassen-q.nav.no/stillinger' som base-URL i stedet, for disse testene.

Reply:
Den er god 👍